configure-cat.bash

```diff
@@ -254,25 +256,10 @@ stop_sonarqube(){
   pkill -u sonarqube
 }
 
-########################################################
-# function custom_server_config
-#
-# Description :
-# Customize SonarQube server configuration
-# 
-################################################################################
-custom_server_config(){
-  # Disable telemetry
-  sed -i 's/#sonar\.telemetry\.enable=true/sonar\.telemetry\.enable=false/' /opt/sonarqube/conf/sonar.properties
```

Comment:
I don't understand why we would not disable the telemetry anymore...
What's the reason for deleting this ?

Reply:
It has been moved to the end of the Dockerfile, because it is a static configuration of the image and do not need to be in the entrypoint (two more static configurations have been added with).